docker-compose.yml

```diff
@@ -198,6 +198,8 @@ services:
       - _APP_DATABASE_SHARED_TABLES_V1
       - _APP_DATABASE_SHARED_NAMESPACE
       - _APP_FUNCTIONS_CREATION_ABUSE_LIMIT
+      - _APP_DATABASE_KEYS
```

Comment:
is this variable needed here? I dont see it used anywhere

Reply:
it is used in projects.php controller

Reply:
It is used only for cloud, so we can remove t